From 765997fa05794e2f71d39351284a354b63df38d0 Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Mon, 18 Aug 2014 19:56:41 -0400
Subject: [scsi] qla2xxx: Enable fw_dump_size for ISP8044

Message-id: <1408391817-6389-54-git-send-email-cdupuis@redhat.com>
Patchwork-id: 87935
O-Subject: [PATCH RHEL 7.1 e-stor 53/69] qla2xxx: Enable fw_dump_size for ISP8044.
Bugzilla: 1089346
RH-Acked-by: Rob Evers <revers@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Hiral Patel <hiral.patel@qlogic.com>

scsi commit id cf3af76b18d8aecfe0fac678ac0fe2eb2d22a135

Signed-off-by: Hiral Patel <hiral.patel@qlogic.com>
Signed-off-by: Saurav Kashyap <saurav.kashyap@qlogic.com>
Signed-off-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/qla2xxx/qla_attr.c b/drivers/scsi/qla2xxx/qla_attr.c
index 27b80e6..e2bd9dd 100644
--- a/drivers/scsi/qla2xxx/qla_attr.c
+++ b/drivers/scsi/qla2xxx/qla_attr.c
@@ -1406,7 +1406,7 @@ qla2x00_fw_dump_size_show(struct device *dev, struct device_attribute *attr,
 
  if (!ha->fw_dumped)
   size = 0;
- else if (IS_QLA82XX(ha))
+ else if (IS_P3P_TYPE(ha))
   size = ha->md_template_size + ha->md_dump_size;
  else
   size = ha->fw_dump_len;
-- 
1.7.1